x-pack/plugins/aiops/server/routes/queries/fetch_terms_2_categories_counts.ts

```diff
@@ -102,8 +106,36 @@ export async function fetchTerms2CategoriesCounts(
         size: 2,
         maxPValue: Math.max(term.pValue ?? 1, category.pValue ?? 1),
         doc_count: 0,
-        support: 1,
-        total_doc_count: 0,
+        support: 0,
+        total_doc_count: Math.max(term.total_doc_count, category.total_doc_count),
+      });
+    });
+
+    itemSets.forEach((itemSet) => {
+      searches.push({ index: params.index });
+      searches.push(
+        getTerm2CategoryCountRequest(
+          params,
+          Object.entries(itemSet.set).map(([fieldName, fieldValue]) => ({
+            fieldName,
+            fieldValue,
+          })),
+          category.fieldName,
+          { key: `${category.key}`, count: category.doc_count, examples: [] },
```

Comment:
There is a bug I've recently discovered where the query created from the category key will not match any documents.
I will create an issue for it, but I don't immediately know how to fix it. 
Basically, from what have seen, if the document has a string that looks like this `foo:bar` the category key might look like this `foo bar`. These will not match in the query.
There is a risk that changing the query will cause it to become more greedy and match documents which aren't in the category.
Using the regex from the category would probably work, but could be very expensive, and may not work at all if the cluster disallows expensive queries.

This comment is just a heads up that you may get 0 counts here, depending on the data.